### PR TITLE
Add core SGX service

### PR DESCRIPTION
**Pull request by omeg**
SGX service with (so far) GSB API for verifying enclave quotes with Intel Attestation Service

### CODE REVIEW

File: exe-unit/src/service/metrics.rs [hunk 1]
```diff
@@ -95,7 +95,7 @@ impl Handler<GetMetrics> for MetricsService {
         let mut metrics = Vec::with_capacity(self.usage_vector.len());
 
         for name in self.usage_vector.iter() {
-            let metric = self
+            /*let metric = self
```
Comment (mfranciszkiewicz) [inline exe-unit/src/service/metrics.rs:98]: How about making this feature enabled, e.g. with `#[cfg(feature = "sgx")]`, `#[cfg(not(feature = "sgx"))]`

Comment (omeg) [reply exe-unit/src/service/metrics.rs:98]: Oops, this commit was included here by mistake.

File: core/model/src/sgx.rs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+pub use graphene::ias::AttestationResponse;
+use serde::{Deserialize, Serialize};
+use ya_service_bus::RpcMessage;
+
+/// Public SGX bus address.
+pub const BUS_ID: &str = "/public/sgx";
```
Comment (mfranciszkiewicz) [inline core/model/src/sgx.rs:6]: Will these endpoints be called via network? If not, bindings could be local

File: core/model/Cargo.toml [hunk 3]
```diff
@@ -35,3 +37,4 @@ log = "0.4"
 serde = { version = "^1.0", features = ["derive"] }
 serde_bytes = "0.11.3"
 thiserror = "1.0.9"
+graphene_api = { git = "https://github.com/golemfactory/graphene-rust.git", rev = "8e14e8f012e790ccce5c696f20c1d750ca3f4398", optional = true }
```
Comment (mfranciszkiewicz) [inline core/model/Cargo.toml:40]: It might be a good idea to split that crate. AFAIU only models are needed here

**Review by mfranciszkiewicz — CHANGES_REQUESTED** [review]

File: exe-unit/src/service/metrics.rs (hunk 1 above)
Comment (omeg) [reply exe-unit/src/service/metrics.rs:98]: Rebased without this commit, it'll be in a separate PR.

File: core/model/src/sgx.rs (hunk 2 above)
Comment (omeg) [reply core/model/src/sgx.rs]: At first I thought it would be needed but in reality this will only be called locally. Changed.

**Review by mfranciszkiewicz — APPROVED** [review]

**Pull request merged**